### PR TITLE
SDIT-1244 Remove unused get appointment endpoints

### PR DESCRIPTION
**Pull request by karenmillermoj**
Removed
- GET /api/bookings/{bookingId}/appointments
- GET /api/bookings/{bookingId}/appointments/today
- GET /api/bookings/{bookingId}/appointments/ThisWeek
- GET /api/bookings/{bookingId}/appointments/nextWeek

### CODE REVIEW

File: src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AppointmentsResourceIntTest.kt [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter.ISO_LOCAL_DATE_TIME
+
+class AppointmentsResourceIntTest : ResourceTest() {
+  @Autowired
+  private lateinit var bookingRepository: BookingRepository
+
+  @Nested
+  @DisplayName("POST /api/bookings/{bookingId}/appointments")
+  inner class CreateAppointments {
```
Comment (karenmillermoj) [inline src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AppointmentsResourceIntTest.kt:22]: Moved create bulk appointments out of cucumber tests

File: src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AppointmentsResourceTest.kt [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+import uk.gov.justice.hmpps.prison.api.model.ScheduledEvent
+import uk.gov.justice.hmpps.prison.api.model.bulkappointments.AppointmentDefaults
+import uk.gov.justice.hmpps.prison.api.model.bulkappointments.AppointmentDetails
+import uk.gov.justice.hmpps.prison.api.model.bulkappointments.AppointmentsToCreate
+import uk.gov.justice.hmpps.prison.repository.BookingRepository
+import java.time.LocalDateTime
+import java.time.temporal.ChronoUnit
+import java.util.Optional
+
+class AppointmentsResourceTest : ResourceTest() {
```
Comment (karenmillermoj) [inline src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AppointmentsResourceTest.kt:35]: Just converted these tests from java to kotlin

File: src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AppointmentsResourceTest.kt [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+  @SpyBean
+  private lateinit var proxyUserAspect: ProxyUserAspect
+
+  @MockBean
+  private lateinit var bookingRepository: BookingRepository
```
Comment (mikehalmamoj) [inline src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AppointmentsResourceTest.kt:40]: Might be able to speed up the intergration tests by making this a SpyBean and moving both spy bean into the superclass.

**Review by mikehalmamoj — APPROVED** [review]

File: src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AppointmentsResourceTest.kt (hunk 3 above)
Comment (karenmillermoj) [reply src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/AppointmentsResourceTest.kt:40]: I have to do another PR for additional tests so will look at that in the next PR

**Pull request merged**